pkg/installer/install.go

```diff
@@ -817,37 +826,54 @@ func copyFile(src, dest string) error {
 	return err
 }
 
-func ensurePrivilegedSCC(client dynamic.Interface, namespace string) error {
-	securityGV, err := schema.ParseGroupVersion("security.openshift.io/v1")
+func createVPNSCC(securityClient securityclient.Interface) (*securityv1.SecurityContextConstraints, error) {
+	securityScheme := runtime.NewScheme()
+	securityCodecs := serializer.NewCodecFactory(securityScheme)
+	securityv1.AddToScheme(securityScheme)
+	obj, err := runtime.Decode(securityCodecs.UniversalDecoder(securityv1.SchemeGroupVersion), vpnSCCBytes)
 	if err != nil {
-		return err
+		panic(fmt.Sprintf("cannot decode static vpn scc: %v", err))
 	}
-	sccGVR := securityGV.WithResource("securitycontextconstraints")
-	obj, err := client.Resource(sccGVR).Get("privileged", metav1.GetOptions{})
-	if err != nil {
-		return err
+	var ok bool
+	scc, ok := obj.(*securityv1.SecurityContextConstraints)
+	if !ok {
+		panic(fmt.Sprintf("static vpn scc is not of the right type: %T", obj))
+	}
+	scc, err = securityClient.SecurityV1().SecurityContextConstraints().Create(scc)
+	if err != nil && !errors.IsAlreadyExists(err) {
+		return nil, fmt.Errorf("failed to create VPN scc: %v", err)
 	}
-	users, exists, err := unstructured.NestedStringSlice(obj.Object, "users")
 	if err != nil {
-		return err
+		scc, err = securityClient.SecurityV1().SecurityContextConstraints().Get("hypershift-vpn", metav1.GetOptions{})
+		if err != nil {
+			return nil, fmt.Errorf("cannot fetch hypershift-vpn scc: %v", err)
+		}
 	}
-	userSet := sets.NewString()
-	if exists {
-		userSet.Insert(users...)
+	return scc, nil
+}
+
+func ensureVPNSCC(securityClient securityclient.Interface, namespace string) error {
+	scc, err := securityClient.SecurityV1().SecurityContextConstraints().Get("hypershift-vpn", metav1.GetOptions{})
+	if err != nil && errors.IsNotFound(err) {
```

Comment:
if !IsNotFound()

Reply:
ha, I could swear I had fixed that locally, thx!